plugins/lighthouse/src/index.ts

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export { default } from './plugin';
+export * from './api';
```

Comment:
rollup outputs a warning for this - it doesn't love mixed default vs. named. I could attach the API stuff to the Plugin itself, I guess, but that feels odd. lmk if you want something different here.

Reply:
I imagine we want to move away from using the default export for plugin packages, export `plugin` or smth like that instead. This is the way atm though, so imo keep it for now.

Reply:
https://github.com/spotify/backstage/issues/523